src/Tribe/RSVP.php

```diff
@@ -1071,18 +1071,13 @@ public function front_end_tickets_form( $content ) {
 		$tickets = $this->get_tickets( $post->ID );
 
 		if ( empty( $tickets ) ) {
-			return;
+			return $content;
 		}
 
-		// Bail on Gutenberg
-		$template_overwrite = tribe( 'tickets.editor.template.overwrite' );
-
+		// test for blocks in content, but usually called after the blocks have been convereted
 		if (
-			tribe_get_option( 'toggle_blocks_editor' )
-			&& (
-				has_blocks( $post->ID )
-				|| ! $template_overwrite->has_classic_editor( $post->ID )
-			)
+			has_blocks( $content )
+			|| false !== strpos( (string) $content, 'tribe-block' )
```

Comment:
Will it ever not have blocks in $content but still have tribe-block?

Reply:
I don't believe so - that is the class we put on blocks.

Reply:
that class does appear in other pages (I'm looking at you AR) but I'm not seeing any issues in my tests.

Reply:
By changing it to `tribe-block ` (with a space at the end) we're specifically targeting blocks and avoiding tripping on things like `tribe-block__rsvp__ticket tribe-block__rsvp__ticket--inactive`